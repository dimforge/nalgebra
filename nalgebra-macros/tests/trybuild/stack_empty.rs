use nalgebra_macros::stack;

fn main() {
    stack![];
}
