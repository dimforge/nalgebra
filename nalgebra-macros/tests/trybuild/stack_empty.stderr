error: `stack` macro cannot be used without any arguments
 --> tests/trybuild/stack_empty.rs:4:5
  |
4 |     stack![];
  |     ^^^^^^^^
  |
  = note: this error originates in the macro `stack` (in Nightly builds, run with -Z macro-backtrace for more info)
